app/mbedtls/app/espconn_mbedtls.c

```diff
@@ -798,7 +796,7 @@ static bool mbedtls_msg_config(mbedtls_msg *msg)
 		mbedtls_ssl_conf_authmode(&msg->conf, MBEDTLS_SSL_VERIFY_NONE);
 	}
 	mbedtls_ssl_conf_rng(&msg->conf, mbedtls_ctr_drbg_random, &msg->ctr_drbg);
-	mbedtls_ssl_conf_dbg(&msg->conf, mbedtls_dbg, NULL);
```

Comment:
Please do not apply this hunk.  See https://github.com/nodemcu/nodemcu-firmware/pull/2214/commits/b3822919d23152a21709492afe93f69e2b858e14 .